frontend/app/src/components/home/communities/edit/ChooseChannels.svelte

```diff
@@ -8,7 +8,7 @@
     import EditableChannel from "./EditableChannel.svelte";
     import ErrorMessage from "../../../ErrorMessage.svelte";
 
-    const MIN_CHANNEL_LENGTH = 4;
```

Comment:
The backend change also includes group and community names

Reply:
For groups it is already 3

Reply:
Community names have a min of 4 on FE

Reply:
I can sort this